From: Ben Hutchings <ben@decadent.org.uk>
Subject: aufs: mark as staging
Forwarded: not-needed

This hasn't been reviewed upstream, so should have the same status
as code in drivers/staging.

--- a/fs/aufs/module.c
+++ b/fs/aufs/module.c
@@ -104,6 +104,7 @@ MODULE_DESCRIPTION(AUFS_NAME
 	" -- Advanced multi layered unification filesystem");
 MODULE_VERSION(AUFS_VERSION);
 MODULE_ALIAS_FS(AUFS_NAME);
+MODULE_INFO(staging, "Y");
 
 /* this module parameter has no meaning when SYSFS is disabled */
 int sysaufs_brs = 1;
